src/jit/codegenxarch.cpp

```diff
@@ -5439,15 +5560,21 @@ void CodeGen::genConsumePutStructArgStk(
         genConsumeReg(srcAddr);
     }
 
-    // If the op1 is already in the dstReg - nothing to do.
-    // Otherwise load the op1 (GT_ADDR) into the dstReg to copy the struct on the stack by value.
+// If the op1 is already in the dstReg - nothing to do.
+// Otherwise load the op1 (GT_ADDR) into the dstReg to copy the struct on the stack by value.
```

Comment:
We'll want to put a CLANG_FORMAT_COMMENT_ANCHOR here or move the comment so it stays indented at the normal indentation.